optimize/backend/src/main/java/io/camunda/optimize/rest/providers/ResponseTimezoneFilter.java

```diff
@@ -26,17 +24,16 @@
 
 @Provider
 @Component
-public class ResponseTimezoneFilter implements ContainerResponseFilter {
```

Comment:
Oh, yes, good point - it is not a regular filter and most of all it will not work with this change. Not sure whether we should make it work first or go without it and fix it during the single app testing as suggested by @RomanJRW